### DISCUSSION TITLE
Installer

### DISCUSSION THREAD

**Discussion by tesivo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#34] [Annunci]
Non ho ancora una data a riguardo, ma, notando il lavoro della traduzione dei nostri cugini francesi di VTMB, e della unofficial patch, anche la nostra traduzione verrà arricchita da installer.

Ciò renderebbe la traduzione quasi ufficiale nell'unofficial patch(ovviamente).

Che ne pensate?

**Comment by NarakuITA** [comment]
Eeeh meee colion. Direi di si! XD ^_^
